assets/src/blocks/Happypoint/HappypointBlock.js

```diff
@@ -26,6 +27,12 @@ export class HappypointBlock {
         },
         iframe_url: {
           type: 'string',
+          validation: ({ iframe_url, mailing_list_iframe }) => {
+            const isValid = mailing_list_iframe === false || iframe_url !== '';
+            const messages = ['Happypoint: "Use mailing list iframe" is set to "On" but there is no IFrame URL'];
```

Comment:
Should we add translations to this message? 🤔

Reply:
Yes! I'll do.